README.md

```diff
@@ -11,39 +11,39 @@ Hier findest du die technischen Präsentationen des Workshops:
+Ab dem Kapitel [What is Django?](https://tutorial.djangogirls.org/en/django/) geht 's los: wir tauchen richtig ins Django Girls Tutorial ein. Dein Django Blog ist ein von deiner statischen Webseite unabhängiges Projekt. Es muss also in einem anderen Ordner gestartet werden. Auch in Github wird es an einem anderen Ort abgelegt als das, was du bis jetzt gemacht hast. Wir werden die Codes immer noch im Github speichern, aber nur als Zwischenschritt. Da Github selbst nur statische Seiten darstellen kann, werden wir für die Darstellung unserer dynamischen Webseite noch einen anderen Service \(Pythonanywhere\) nutzen. Das klingt nicht nur kompliziert, sondern ist es auch. Deine dynamische Seite online zu bringen ist eine grosse Leistung. Aber keine Sorge, es ist alles detailliert im Django Girls Tutorial beschrieben und wir werden dir dabei helfen!
 
 Nun ab ins Tutorial --&gt; [What is Django?](https://tutorial.djangogirls.org/en/django/)!
 
 ### C. Extras
 
-Je nach Zeit und Interesse kannst du während des Workshops, zu Hause oder an einem der Follow up Events an folgenden Sachen weiter arbeiten:
+Je nach Zeit und Interesse kannst du während des Workshops, zu Hause oder an einem der Follow up-Events an folgenden Sachen weiter arbeiten:
 
 * [Deine statische Seite in Django einbinden](./extras/adding-your-static-page-to-django.md)
   * [What's next?](https://tutorial.djangogirls.org/en/whats_next/)
 * Mit den [Django Girls Tutorial Extensions](https://djangogirls.gitbooks.io/django-girls-tutorial-extensions/content/) deinen Blog noch besser machen
 
 > Dieses Tutorial ist lizenziert unter der Creative Commons Attribution-ShareAlike 4.0 International Lizenz. Gehe auf [https://creativecommons.org/licenses/by-sa/4.0/](https://creativecommons.org/licenses/by-sa/4.0/) um eine Kopie der Lizenz zu sehen.
 >
-> Das Tutorial ist gemacht um in Zusammenspiel mit dem offiziellen Django Girls Tutorial bearbetiet zu werden und hat aus für die Zweckmässigkeit einige Teile davon dupliziert. \([https://tutorial.djangogirls.org/en](https://tutorial.djangogirls.org/de), lizenziert unter der Creative Commons Attribution-ShareAlike 4.0 International Lizenz\).
+> Das Tutorial ist gemacht, um im Zusammenspiel mit dem offiziellen Django Girls Tutorial bearbeitet zu werden und hat aus Zweckmässigkeit einige Teile davon dupliziert. \([https://tutorial.djangogirls.org/en](https://tutorial.djangogirls.org/de), lizenziert unter der Creative Commons Attribution-ShareAlike 4.0 International Lizenz\).
```

Comment:
Den unbestimmten Artikel ("in Zusammenspiel" anstatt "im Zusammenspiel") hättest du belassen können. Aber mit dem Bestimmten ("im") ist es auch korrekt.

Deine anderen Korrekturen hier brauchts es natürlich. :+1: